package.json

```diff
@@ -51,6 +51,7 @@
 		"db-migrate": "^0.11.14",
 		"del-cli": "^5.1.0",
 		"eslint": "^8.50.0",
+		"eslint-config-prettier": "^9.1.0",
```

Comment:
After this diff is merged, please mention in `#launch-module-web-wizards` that the team will need to re-run `npm install`